src/pip/_internal/operations/install/legacy.py

```diff
@@ -73,7 +73,16 @@ def install(
             )
 
         if not os.path.exists(record_filename):
-            logger.debug('Record file %s not found', record_filename)
+            logger.warning('Record file %s not found', record_filename)
+            deprecated(
```

Comment:
This will only show up for the first package, but I think that's probably OK since the warning will show up for all of them.

Reply:
Yea, I spent some time thinking about this and printing on the last requirement may be more valuable?

I couldn't come up with any good way to deal with this; so suggestions are welcome.

Reply:
No suggestion, OK as-is for me.